_source/_includes/legacy-api.html

```diff
@@ -0,0 +1,17 @@
+<div class="alert alert-info" role="alert">
+  <p class="title">
+    <a class="close" aria-label="Close" title="Close" onclick="this.parentElement.parentElement.style.display='none'">
+      <span aria-hidden="true">&times;</span>
+    </a>
+    Heads up... This blog post is old!
+  </p>
+  <p>This post is designed to work with the older Okta legacy API. If you're a
+    new to Okta
```

Comment:
This currently says "If you're a new to Okta". The "a" should be removed. However, I can't help but think it'd be better to give a particular date rather than just "new". Something like "if you created your Okta developer account after Feb 1, 2021...". 

Other thoughts:

- We might want to also give instructions for how to upgrade to an OIE account.
- Will it be possible to create an "old" account? What if someone really wants to do the tutorial, but they just created an account?

Reply:
I'll fix that!

RE: putting phrasing in there about the date -- I'm waiting to find out what all that messaging will be from Product before this goes live for reals. So it'll change for sure. This is just a placeholder until they release 'official' messaging.

RE: instructions on how to create an OIE account, they will be creating that and giving us something to link to. Will put that in when it's finished.

RE: will it be possible to create an older account: no :( They are not going to allow it.